pandas/tests/test_generic.py

```diff
@@ -1859,6 +1859,26 @@ def test_pipe_panel(self):
         with tm.assertRaises(ValueError):
             result = wp.pipe((f, 'y'), x=1, y=1)
 
+    def test_pct_change_frame(self):
+        pnl = DataFrame([np.arange(0, 40, 10), np.arange(0, 40, 10), np.arange(0, 40, 10)]).astype(np.float64)
+        pnl.iat[1,0] = np.nan
+        pnl.iat[1,1] = np.nan
+        pnl.iat[2,3] = 60
+        
+        mask = pnl.isnull()
+
+        expected_axis0 = pnl.ffill(axis=0)/pnl.ffill(axis=0).shift(axis=0) - 1
+        expected_axis0[mask] = np.nan
+        result_axis0 = pnl.pct_change(axis=0, fill_method='pad')
+
+        self.assert_frame_equal(result_axis0, expected_axis0)
+
```

Comment:
what do you mean ? There is no ``test_frame`` and I made the only one ``test_pct_change_frame`` test